src/NBitcoin/Network.cs

```diff
@@ -235,19 +235,20 @@ public virtual Consensus Clone()
 
     public partial class Network
     {
-        private uint magic;
+        private static readonly ConcurrentDictionary<string, Network> NetworksContainer = new ConcurrentDictionary<string, Network>();
+
         private byte[] alertPubKeyArray;
         private PubKey alertPubKey;
-        private readonly List<DNSSeedData> seeds;
-        private readonly List<NetworkAddress> fixedSeeds;
-        private readonly Dictionary<int, CheckpointInfo> checkpoints = new Dictionary<int, CheckpointInfo>();
         private Block genesis;
-        private Consensus consensus = new Consensus();
-
+        
         private Network()
         {
-            this.seeds = new List<DNSSeedData>();
-            this.fixedSeeds = new List<NetworkAddress>();
+            this.DNSSeeds = new List<DNSSeedData>();
```

Comment:
Can you please add comments for those properties? (I know it's kind of not the aim of this PR but if not in this PR- no one will do it for months)

Reply:
yeah you're right. 
I was thinking this class' properties are some of the most important ones we have in the code. Every user using our node will have to understand what they mean and what they're for. That's also why I didn't pay attention to them at this stage.
I'll do that in part 2 if that's ok, I really want to get going on the Integration Tests.